cmd/eksctl-anywhere/cmd/validatecreatecluster.go

```diff
@@ -0,0 +1,140 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/types"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/validations/commandvalidations"
+	"github.com/aws/eks-anywhere/pkg/validations/createvalidations"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type validateOptions struct {
+	clusterOptions
+	hardwareCSVPath       string
+	tinkerbellBootstrapIP string
+}
+
+var valOpt = &validateOptions{}
+
+var validateCreateCmd = &cobra.Command{
```

Comment:
I think we need a another command?
Looking at this file `validatecreatecluster`, shouldn't the command be exp -> validate -> create -> cluster?

Reply:
updated to use `exp validate create cluster`